# Generated for NaoQi method ALVisionToolbox.halfPress
# Prepare camera for shooting (like the auto-focus on standard and digital cameras)

---


